### ISSUE TITLE
Filter Key Failure

### ISSUE THREAD

**Issue by joleenf** [ssec/polar2grid#315]
@djhoese I encountered an error when running the mirs reader with updates from master branch (including filters).  The error is due to a "day_only" KeyError:
```python
Traceback (most recent call last):
  File "/Users/joleenf/code/polar2grid/polar2grid/glue.py", line 550, in <module>
    sys.exit(main())
  File "/Users/joleenf/code/polar2grid/polar2grid/glue.py", line 519, in main
    scn = filter_scene(scn, reader_names,
  File "/Users/joleenf/code/polar2grid/polar2grid/filters/_filter_scene.py", line 91, in filter_scene
    criteria = get_reader_filter_criteria(reader_names, 'day_only')
  File "/Users/joleenf/code/polar2grid/polar2grid/filters/_filter_scene.py", line 52, in get_reader_filter_criteria
    readers_criteria = [_get_single_reader_filter_criteria(reader, filter_name)
  File "/Users/joleenf/code/polar2grid/polar2grid/filters/_filter_scene.py", line 52, in <listcomp>
    readers_criteria = [_get_single_reader_filter_criteria(reader, filter_name)
  File "/Users/joleenf/code/polar2grid/polar2grid/filters/_filter_scene.py", line 47, in _get_single_reader_filter_criteria
    return filter_criteria[filter_name]
KeyError: 'day_only'
```

Perhaps adding:

```python
FILTERS = { 'day_only': {}, 'night_only': {}, }
```

Is the only fix that is needed for the mirs reader?  However, there are probably other readers that will produce this error without the FILTERS line?

**Comment by djhoese** [comment]
I ran into this too. I'm fixing it in #314

**Comment by djhoese** [comment]
Closed by #314.
